src/node_modules/graph-view/node-ui.js

```diff
@@ -65,29 +62,48 @@ module.exports = React.createClass({
     // var roleIcons = (activeRoleTypes.length > 0) ? this.renderRoleIcons(activeRoleTypes, r) : null
 
     // debug('node', this.props)
+    var isPerson = d.model.getType() === 'Person'
 
     return (
       React.createElement('g', { className: 'node' }, [
-        React.createElement('circle', {
-          cx: x+d.w/2,
-          cy: y+d.w/2,
-          r: d.w/2+2,
+        isPerson ? React.createElement('circle', {
+          cx: d.x,
+          cy: d.y,
+          r: Math.max(d.w / 2) + 2,
           fill: '#ECEFF1'
-        }),
+        }) : null /* React.createElement('rect', {
```

Comment:
@simontegg i'm torn whether we should have a rectangular background behind a group avatar similar to the people avatar. in some ways it makes sense, as it emphasizes the 'group as square' symbol, but also is blocky and sometimes the arrows are behind the rectangle (which is a bug anyways for groups with square icons), so for now i just commented it out.